test/integration/items_store.go

```diff
@@ -1,4 +1,4 @@
-package mgo
+package integration
```

Comment:
What's the reason to move the test to the integration package? Ideally, mgo and mongo integration tests would live in different packages.